STYLE_GUIDE.md

```diff
@@ -0,0 +1,97 @@
+# panther-analysis Style Guide
+
+## Metadata Best Practices
+
+### RuleID, Filename and DisplayName
+
+- RuleID, Filename and DisplayName should be similar enough to eachother to be able to easily identify a filename by it's RuleID or vice versa.
+- RuleID should start with the LogType idenifier followed by a `.`
+
+```yaml
+DisplayName: "AWS Compromised IAM Key Quarantine"
+RuleID: "AWS.CloudTrail.IAMCompromisedKeyQuarantine"
+Filename: aws_iam_compromised_key_quarantine.py
+```
+
+### Severity
+
+Review the [Alert Severity Guidelines](https://docs.panther.com/detections/rules#alert-severity) in Panther's documentation.  Consider additional factors that could increase or decrease severity, such as exploitation in the wild, potential for false positives, and actionability.
+
+### Reference
+
+Reference should link to a relevant security research or threat report describing the attack being detected and why you'd want to detect it from a security perspective.  Links to API docs or other generic log source documentation should be avoided.
+
+### Runbook
+
+Runbooks should provide clear triage steps for incident responders.  See [Define Clear Triage Steps](https://jacknaglieri.substack.com/i/148126819/define-clear-triage-steps).
+
+### MITRE ATT&CK Tactics and Techniques
```

Comment:
you could consider making this "Reports" since the report name can be anything I think, but I also understand that Mitre is the most commonly used